src/components/header/Topbar.tsx

```diff
@@ -10,7 +10,7 @@ import Logo from '../navigation/Logo';
 
 type TopbarProps = {
     isNavigationOpen: boolean;
-    onNavigationToggle: Function;
+    onNavigationToggle?: Function;
```

Comment:
We don't need this function on the login page and previously an empty function was being passed... seems like a foolish pattern to me.